### PR TITLE
Gets plugin ready for PCT

### PR DESCRIPTION
**Pull request by alecharp**
There is a situation which can only be reproduced in the context of testing this plugin on a test instance where an implementation of this API is available. Basically, if we add any `display-url-api` implementation plugin in the test dependencies of `display-url-api`.

In the case of https://github.com/jenkinsci/bom/pull/3137, we need to run the tests of `display-url-api` plugin while there is an implementation of `DisplayURLProvider`. Because of this, the tests which were "expecting" an implementation (without configuring it) were failing.
By configuring them, we bypass the automatic selection in https://github.com/jenkinsci/display-url-api-plugin/blob/d3e2e182cdfb9919b122813c827be9d6e929685e/src/main/java/org/jenkinsci/plugins/displayurlapi/DisplayURLProvider.java#L221-L223, but we ensure what is tested.

This is related to https://github.com/jenkinsci/bom/issues/3136 and is made to fix https://github.com/jenkinsci/bom/pull/3137.

### Testing done

Forcing `pipeline-graph-view` plugin in the test dependencies, I ran `mvn verify` locally to validate the modifications.

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <properties>
     <revision>2</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.361.1</jenkins.version>
+    <jenkins.version>2.440.1</jenkins.version>
```
Comment (MarkEWaite) [inline pom.xml:19]: Can you explain further why the minimum Jenkins version was increased, especially to so recent a version?
```suggestion
    <jenkins.version>2.361.1</jenkins.version>
```

File: pom.xml [hunk 2]
```diff
@@ -62,8 +62,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.361.x</artifactId>
-        <version>1836.vfe602c266c05</version>
+        <artifactId>bom-2.440.x</artifactId>
+        <version>2982.vdce2153031a_0</version>
```
Comment (MarkEWaite) [inline pom.xml:66]: Tests pass without requiring a newer Jenkins minimum version (at least on my Linux computer).  Is there something that prompts the change to a newer Jenkins minimum version when this is an API plugin and would generally keep a much older minimum Jenkins version?
```suggestion
        <artifactId>bom-2.361.x</artifactId>
        <version>1836.vfe602c266c05</version>
```

**Review by MarkEWaite — COMMENTED** [review]
Changes look good to me, though I'd like to better understand why the minimum Jenkins version was increased.  I think it should stay at 2.361.1 because this is an API plugin and we prefer to keep their minimum Jenkins version older.

**Comment by alecharp** [comment]
@MarkEWaite You are absolutely correct: the `jenkins.version` bump was unnecessary, I reverted it in https://github.com/jenkinsci/display-url-api-plugin/pull/212/commits/7d67f463c81ca7041a463215b37d94cf94bc24e6.

I validated those changes running the `local-test.sh` in BOM, tweaking the `pct.sh` script to use my local checkout dir of `display-url-api` and all tests passed.

**Comment by alecharp** [comment]
(click wrong button)

**Review by MarkEWaite — APPROVED** [review]

**Review by car-roll — APPROVED** [review]

**Pull request merged**